apps/examples/testcase/ta_tc/device_management/utc/utc_dm_main.c

```diff
@@ -40,6 +51,52 @@ extern int working_tc;
 
 static int isConnected = 0;
 
+static slsi_security_config_t *get_security_config(char *sec_type, char *psk)
```

Comment:
Hi, @dr-venkman 
I have a question. The 'security_config' function already exists. Why did you create a new one?
https://github.com/Samsung/TizenRT/blob/master/apps/netutils/wifi/slsi_wifi_utils.c#L34
`getSecurityConfig`